docs/resources/azurerm_virtual_networks.md.erb

```diff
@@ -0,0 +1,103 @@
+---
+title: About the azurerm_virtual_networks Resource
+platform: azure
+---
+
+# azurerm\_virtual\_networks
+
+Use the `azurerm\_virtual\_networks` InSpec audit resource to test properties related to
+virtual networks for a resource group.
+
+<br />
+
+## Availability
+
+### Installation
+
+This resource is available in the `inspec-azure` [resource
+pack](https://www.inspec.io/docs/reference/glossary/#resource-pack). To use it, add the
+following to your `inspec.yml` in your top-level profile:
+
+    depends:
+      inspec-azure:
+        git: https://github.com/inspec/inspec-azure.git
+
+You'll also need to setup your Azure credentials; see the resource pack
+[README](https://github.com/inspec/inspec-azure#inspec-for-azure).
+
+### Version
+
+This resource first became available in 1.0.0 of the inspec-azure resource pack.
+
+## Syntax
+
+The `resource_group` must be given as a parameter.
+
+    describe azurerm_virtual_networks(resource_group: 'MyResourceGroup') do
+      ...
+    end
+
+## Examples
+
+    # Exists if any virtual networks exist in the resource group
+    describe azurerm_virtual_networks(resource_group: 'MyResourceGroup') do
+      it { should exist }
+    end
+<br />
+
+## Parameters
+
+  - `resource_group`
+
+### Parameter Examples
+
+### resource\_group (required)
+
+    describe azurerm_virtual_networks(resource_group: 'MyResourceGroup') do
+      ...
+    end
+
+<br />
+
+### name
+
+Filters the result to only those that match the given name.
+
+    # Insist that you have at least one virtual network that starts with 'prefix'
+    describe azurerm_virtual_networks(resource_group: 'MyResourceGroup').where { name.start_with?('prefix') } do
+      it { should exist }
+    end
```

Comment:
I wonder if this should be a second example, after one showing a `name == 'SomeName'` filter. "Starts with" is interesting, but the description above implies equality with "match the given name" and this feels inconsistent.

Reply:
No problem.  I just took that from the virtual_machine doc.

Reply:
I added a second example.